src/config.js

```diff
@@ -29,7 +29,7 @@ global.config = {
 
   info: {
     signController: true,
-    signText: 'Fully automated TooAngel bot: https://github.com/TooAngel/screeps',
+    signText: 'Fully automated TooAngel bot: http://tooangel.github.io/screeps/',
```

Comment:
The page is created from the README.md. The documentation gets improved with #197 . Should hopefully also fix these issues.